apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -245,11 +246,14 @@ export async function addEventTypesToDb(
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     schedule?: any;
     metadata?: any;
+    team?: { id?: number | null; bookingLimits?: IntervalLimit };
   })[]
 ) {
   log.silly("TestData: Add EventTypes to DB", JSON.stringify(eventTypes));
   await prismock.eventType.createMany({
-    data: eventTypes,
+    data: eventTypes.map((et) => ({
+      ...et,
+    })),
```

Comment:
Is spreading necessary here?

Reply:
Removed that, I think I had something else there before. Thank. you!